docker/data-science/notebooks/ch07/2.semantic-search.ipynb

```diff
@@ -852,175 +868,167 @@
+       "   'longitude_d': -80.695902,\n",
+       "   '_version_': 1794883425504591872},\n",
+       "  {'id': '8ldY6-AZ5uRocBCzhaYtRQ',\n",
+       "   'name_t': 'Mattress Firm Stonecrest at Piper Glen',\n",
+       "   'city_t': 'Charlotte',\n",
+       "   'state_t': 'NC',\n",
+       "   'text_t': 'Bought our mattress here- customer service was rock solid and helpful.  All good!  Would buy from them again.  Free delivery',\n",
+       "   'stars_i': 5,\n",
+       "   'categories_t': 'Home Decor, Home & Garden, Shopping, Mattresses, Furniture Stores',\n",
+       "   'doc_type': ['Home Decor, Home & Garden, Shopping, Mattresses, Furniture Stores'],\n",
+       "   'location_pt_s': '35.0591334,-80.8130775',\n",
+       "   'location_p': '35.0591334,-80.8130775',\n",
+       "   'type_ss': ['Home Decor, Home & Garden, Shopping, Mattresses, Furniture Stores'],\n",
+       "   'latitude_d': 35.0591334,\n",
+       "   'longitude_d': -80.8130775,\n",
+       "   '_version_': 1794883425544437760},\n",
+       "  {'id': 'rpK6XXAfFZIVb2amLgjDDg',\n",
+       "   'name_t': \"Carolina Buyer's Agent\",\n",
+       "   'city_t': 'Charlotte',\n",
+       "   'state_t': 'NC',\n",
+       "   'text_t': \"This is our 5th house buy, and Julie is the best agent we have ever used.  I wouldn't hesitate to recommend her to everyone. \\n\\nWe are new to the area, so her attention to detail and knowledge of North Carolina real estate law saved us money and time, right up to a mistake she caught at closing that put over $600 right back in our pocket.\\n\\nJulie is warm, savvy, and a pleasure to work with.  She was available everytime we called, and proactive if she saw potential problems, never hesitating to go directly to the seller to advocate for us.\\n\\nShe listened to our wants and needs to help us find not only the perfect home but the perfect neighborhood.  I trust Julie 100%, and can't thank her enough for a painless experience, at least on our part.  We love our new home!\",\n",
+       "   'stars_i': 5,\n",
+       "   'categories_t': 'Real Estate Services, Real Estate, Home Services, Real Estate Agents',\n",
+       "   'doc_type': ['Real Estate Services, Real Estate, Home Services, Real Estate Agents'],\n",
+       "   'location_pt_s': '35.2146586,-80.8599193',\n",
+       "   'location_p': '35.2146586,-80.8599193',\n",
+       "   'type_ss': ['Real Estate Services, Real Estate, Home Services, Real Estate Agents'],\n",
+       "   'latitude_d': 35.2146586,\n",
+       "   'longitude_d': -80.8599193,\n",
+       "   '_version_': 1794883425547583488}]}"
       ]
      },
-     "execution_count": 17,
+     "execution_count": 26,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
     "reviews_collection = engine.get_collection(\"reviews\")\n",
-    "request = {\"query\": query_string}\n",
-    "reviews_collection.search(request)"
+    "reviews_collection.search(query=query_string, query_parser=\"default\")"
```

Comment:
I'm assuming this is because the edismax is the new default and it doesn't handle the generated Solr syntax well?

Reply:
In this case that's correct. 

I've gone ahead and implemented default params for most schemas, reviews didn't need these defaults, and thus the "default" query_parser type logic has been removed.